script_library/deployment-actions-openstack.sh

```diff
@@ -64,12 +65,14 @@ function clean_k8s(){
         ansible -m script -a "script_library/cleanup-k8s.sh" osh-deployer -i inventory-osh.ini
     fi
 }
-function clean_openstack(){
-    echo "Deleting on OpenStack"
-    ${socok8s_absolute_dir}/4_osh_node_on_openstack/delete.sh
+function delete_caasp(){
     echo "Delete Caasp nodes"
     ${socok8s_absolute_dir}/3_caasp_nodes_on_openstack_heat/delete.sh || true
     ${socok8s_absolute_dir}/3_caasp_nodes_on_openstack_manually/delete.sh || true
+}
+function clean_openstack(){
+    echo "Deleting on OpenStack"
```

Comment:
couldn't that clean_openstack call delete_caasp as first step instead then?

Reply:
Well, I wanted to split those, so `delete_caasp` is called from `teardown` (although maybe in the wrong order?)